src/main/java/frc/robot/Constants.java

```diff
@@ -16,6 +16,35 @@
  * constants are needed, to reduce verbosity.
  */
 public final class Constants {
-    public static final int intakeMotorPort = 0;
-    public static final int intakeSolenoidPort = 0;
+    
+    public final class Drivetrain {
+        //controllers
+        public final static int driveController = 0;
+        public final static double deadzone = 0.05;
+
+        //drive
+        public final static boolean useLinerAcceleration = true;
+        public final static double accelerationSpeed = 0;
+        public final static int driveTurnAxis = 0;
+        public final static int driveMainAxis = 0;
+        public final static int driveTalonLeftMotor1 = 0;
+        public final static int driveTalonLeftMotor2 = 0;
+        public final static int driveTalonRightMotor1 = 0;
+        public final static int driveTalonRightMotor2 = 0;  //0 should be replaced
+
+        //encoders
+        public final static double encoderTicks = 2048.0;   //encoder ticks per wheel rotation is 2048
+        public final static double wheelDiameter = 6.0;
+        public final static double driveEncoderToInches = 1 / wheelDiameter * Math.PI / encoderTicks;   //makes number inches
```

Comment:
First, did you check the gear ratios with the build team to make sure this is accurate? 
Furthermore, when I multiply your constant by 2048 I should get the distance covered by one revolution of the wheel.
![CodeCogsEqn](https://user-images.githubusercontent.com/26718624/73368316-5fdb4080-4276-11ea-9aba-048ac2c69b13.gif)
Check this out and change it so that it is accurate.